README.md

```diff
@@ -43,6 +43,7 @@ The `version` input parameter specifies the version of Lava to use.
 
 The `config` input parameter specifies the path of the configuration
 file passed to Lava.
+
```

Comment:
In Lava we try to use [semantic linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/). Please, revert. 

```suggestion
```